core/src/SYCL/Kokkos_SYCL_Instance.cpp

```diff
@@ -166,7 +166,7 @@ int SYCLInternal::acquire_team_scratch_space() {
   return current_team_scratch;
 }
 
-sycl::device_ptr<void> SYCLInternal::resize_team_scratch_space(
+Kokkos::Impl::sycl_device_ptr<void> SYCLInternal::resize_team_scratch_space(
```

Comment:
You don't need the `Kokkos::Impl::` qualification?
Same question everywhere

Reply:
Yes, I do. I only qualified where necessary. `Kokkos_SYCL_Instance.[cpp|hpp]` implementations are in a `Kokkos::Experimental::Impl` namespace. As soon as we remove `Experimental` from the `SYCL` backend, it wouldn't be necessary anymore.

Reply:
Then I need to qualify the other half of the occurrences.